pkg/appconsts/appconsts.go

```diff
@@ -1,16 +1,15 @@
 package appconsts
 
 import (
-	"github.com/celestiaorg/nmt/namespace"
 	"github.com/celestiaorg/rsmt2d"
 	"github.com/tendermint/tendermint/pkg/consts"
 )
 
 // These constants were originally sourced from:
 // https://github.com/celestiaorg/celestia-specs/blob/master/src/specs/consensus.md#constants
 const (
-	// NamespaceSize is the namespace size in bytes.
-	NamespaceSize = consts.NamespaceSize
+	// NamespaceSize is the size of a namespace (version + ID) in bytes.
+	NamespaceSize = 33
```

Comment:
I can move consts from pkg/namespace here if that's what you're asking. I don't want this package to depend on pkg/namespace because that could lead to cyclic imports.